modules/nixos/fcitx5.nix

```diff
@@ -0,0 +1,31 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  inherit (config.catppuccin) sources;
+  cfg = config.i18n.inputMethod.fcitx5.catppuccin;
+  enable =
+    cfg.enable
+    && (
+      config.i18n.inputMethod.enable or true
+      && config.i18n.inputMethod.type or config.i18n.inputMethod.enabled == "fcitx5"
+    );
+
+  theme = pkgs.runCommand "catppuccin-fcitx5" { } ''
+    mkdir -p $out/share/fcitx5/themes/
+    cp -r ${sources.fcitx5}/src/catppuccin-${cfg.flavor}/ $out/share/fcitx5/themes/
```

Comment:
the path of the themes got updated as well and is `catppuccin-${cfg.flavor}-${cfg.accent}` now